airbyte-integrations/connectors/source-bing-ads/source_bing_ads/schemas/campaign_performance_report.json

```diff
@@ -38,6 +38,9 @@
     "CampaignStatus": {
       "type": ["null", "string"]
     },
+    "CampaignLabels": {
+      "type": ["null", "string"]
```

Comment:
Could you please confirm that CampaignLabels is just a string, because name assumes that there are a lot of labels (probably list of strings? )

Reply:
@trowacat can you confirm the above, please? thanks

Reply:
From: https://learn.microsoft.com/en-us/advertising/reporting-service/campaignperformancereportcolumn?view=bingads-13#campaignlabels

**CampaignLabels**
The labels applied to the campaign.Labels are delimited by a semicolon (;) in the report download.

Reply:
@trowacat did you have time to take a look on this?

Reply:
@midavadim I added some labels to our integration account data
<img width="804" alt="image" src="https://user-images.githubusercontent.com/5154322/236948514-862475ea-2d2d-4dbd-a528-061999946d4f.png">
The data field is correct.